# Message buffer for transmitting data

uint8[] tx_string